Detectors/ITSMFT/ITS/reconstruction/src/Clusterer.cxx

```diff
@@ -154,15 +156,15 @@ void Clusterer::finishChip(TClonesArray &clusters)
     x = mX0 + x*mPitchX;
     z /= npix;
     z = mZ0 + z*mPitchZ;
-    Cluster &c = *(new (clusters[clusters.GetEntriesFast()]) Cluster());
-    c.setVolumeId(mChipID);
-    c.setX(x);
-    c.setY(0);
-    c.setZ(z);
-    c.setSigmaY2(sigmaX2);
-    c.setSigmaZ2(sigmaY2);
-    c.setNxNzN(xmax-xmin+1,zmax-zmin+1,npix);
-    c.setFrameLoc();
-    c.setLabel(mPreClusterHeads[i1].second, 0);
+    Cluster *c = (Cluster *)clusters.ConstructedAt(noc++);
```

Comment:
how about:
Cluster *c = static_cast<Cluster *>(clusters.ConstructedAt(noc++));

@sawenzel do we have a check in the code checker for c-style casts? If not: shouldn't we?